### PR TITLE
SLING-7169

### PR DESCRIPTION
**Pull request by volteanu**

### CODE REVIEW

**Comment by rombert** [comment]
@volteanu  - you seem to include the changes from #5 and #6 . Can you push a review with just the individual changes for SLING-7169? This would make it easier to review.

File: src/test/resources/simplelogger.properties [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+#
+#  Unless required by applicable law or agreed to in writing,
+#  software distributed under the License is distributed on an
+#  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#  KIND, either express or implied.  See the License for the
+#  specific language governing permissions and limitations
+#  under the License.
+#
+
+org.slf4j.simpleLogger.defaultLogLevel=info
```
Comment (rombert) [inline src/test/resources/simplelogger.properties]: Isn't this the default already?

Comment (volteanu) [reply src/test/resources/simplelogger.properties]: Yes, but it's useful to easily change when you want to debug tests. Should I delete it?

Comment (rombert) [reply src/test/resources/simplelogger.properties]: I would prefer not adding the file as we tend to get classpath clashes. You can always use a CLI argument for it: `-Dorg.slf4j.simpleLogger.defaultLogLevel=debug`

**Comment by rombert** [comment]
Thanks @volteanu for the PR.

**Pull request merged**